.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Why do we need this @0m4r?

Reply:
this is to avoid the warning about peer-dependencies, if I remove that line I get this sort of output when I do `npm install` type of commands:
```shell
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: airbnb-prop-types@2.16.0
npm warn Found: react@18.2.0
npm warn node_modules/react
npm warn   dev react@"^18.2.0" from the root project
npm warn   3 more (react-dom, react-outside-click-handler, use-immer)
npm warn
npm warn Could not resolve dependency:
npm warn peer react@"^0.14 || ^15.0.0 || ^16.0.0-alpha" from airbnb-prop-types@2.16.0
npm warn node_modules/airbnb-prop-types
npm warn   airbnb-prop-types@"^2.15.0" from react-outside-click-handler@1.3.0
npm warn   node_modules/react-outside-click-handler
npm warn
npm warn Conflicting peer dependency: react@16.14.0
npm warn node_modules/react
npm warn   peer react@"^0.14 || ^15.0.0 || ^16.0.0-alpha" from airbnb-prop-types@2.16.0
npm warn   node_modules/airbnb-prop-types
npm warn     airbnb-prop-types@"^2.15.0" from react-outside-click-handler@1.3.0
npm warn     node_modules/react-outside-click-handler
npm warn deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm warn deprecated eslint-plugin-typescript@0.14.0: Deprecated: Use @typescript-eslint/eslint-plugin instead
```